config_files/public/local_config.ini

```diff
@@ -21,14 +21,15 @@ enable_simulator_mode                        = false
 sempoadmin_emails                            = admin@acme.org,example@example.com
 default_country                              = AU
 third_party_sync_epoch                       = latest
-verify_third_party_sync                      = true
+verify_third_party_sync                      = false
```

Comment:
A bit out of scope, but it doesn't typically help to have this enabled when running locally